doc: add doc for osm-to-railjson (#344)
DGEXSolutions/osrd-website: content/docs/guides/osm_infra/_index.en.md

---

> title: "Generate OpenStreetMap infrastructure"

Reviewer: ```suggestion
title: "OpenStreetMap infrastructures"
```

Author: 6a016b4612d0c2dcad8205d6d547e91a1a4f5606

> linkTitle: "Generate OpenStreetMap infrastructure"

Reviewer: ```suggestion
linkTitle: "OpenStreetMap infrastructures"
```

Author: 6a016b4612d0c2dcad8205d6d547e91a1a4f5606

weight: 10

> description: Generate infrastructure with OSM open data with our [osm-to-railjson](https://github.com/OpenRailAssociation/osrd/tree/dev/editoast/osm_to_railjson) tool

Reviewer 2: ```suggestion
description: Generate an infrastructure using open data from OSM with our [osm-to-railjson](https://github.com/OpenRailAssociation/osrd/tree/dev/editoast/osm_to_railjson) tool
```

---

> As detailed [here](https://osrd.fr/en/docs/explanation/models/data-models-full-example/), in order to utilise all OSRD functions, the following data is required:

Reviewer 2: ```suggestion
As detailed [here](https://osrd.fr/en/docs/explanation/models/data-models-full-example/), in order to benefit from all OSRD functionalities, the following objects are required:
```

 
- Track sections
- Switches
- Curves and slopes
- Signals and detectors
- Routes

> - Electrification

Reviewer 2: ```suggestion
- Electrifications
```

- Operational points

> - Loading gauge

Reviewer 2: ```suggestion
- Loading gauges
```

- Speed sections

> At SNCF, we source this data in private databases and transform it into the railjson format using closed-source code.

Reviewer 2: ```suggestion
For SNCF Réseau production environments, this data is sourced from internal private databases and is then transformed into the railjson format using closed-source code.
```

> If you are a railway infrastructure manager who owns this data for your area of operation, you can do the same.

Reviewer 2: ```suggestion
Any other railway infrastructure manager who owns this data for its own area of operation, this can also be done.
```

> Any other railway infrastructure manager who owns this data for its own area of operation, this can also be done.

Reviewer: ```suggestion
For another railway infrastructure manager owning this data for its own area of operation, this can be done as well.
```

Author: 6a016b4612d0c2dcad8205d6d547e91a1a4f5606

> However, to let anyone (including people outside of railway infrastructure management) try and improve OSRD, we have developed an algorithm that generates realistic (but not real) infrastructure using OpenStreetMap tracks as only input.

Reviewer 2: ```suggestion
However, to let anyone (including people outside of the railway infrastructure industry) try and improve OSRD, we have developed an algorithm that generates a realistic infrastructure using OpenStreetMap tracks as only input. It is important to note that this infrastructure is realistic and not real.
```

This allows to easily generate a real-looking infrastructure for any place in the world!

> As a demonstration, we have put generated intrastructure of France, Belgium and Morocco in our public [demo website](demo.osrd.fr).

Reviewer 2: ```suggestion
As a demonstration, such generated intrastructures (France, Belgium and Morocco) are available in the public [demo website](demo.osrd.fr).
```

> As a demonstration, such generated intrastructures (France, Belgium and Morocco) are available in the public [demo website](https://demo.osrd.fr).

Reviewer 2: ```suggestion
As a demonstration, such generated infrastructures (France, Belgium and Morocco) are available in the public [demo website](https://demo.osrd.fr).
```
